python-avd/scripts/build_schemas.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+import logging
+from pathlib import Path
+from sys import argv, path
+
+# Override global path to load schema from source instead of any installed version.
+path.insert(0, str(Path(__file__).parents[1]))
+
+from schema_tools.build_schemas import build_schemas
+
+if __name__ == "__main__":
+    log_level_str = argv[1].upper() if len(argv) > 1 else "INFO"
```

Comment:
```suggestion
    log_level_str = argv[1].upper() if len(argv) > 1 else "DEBUG"
```

same for the other and clean pre-commit